package.json

```diff
@@ -104,7 +105,9 @@
     "typescript": "^4.5.3"
   },
   "dependencies": {
-    "commander": "^10.0.0"
+    "commander": "^10.0.0",
+    "isomorphic-dompurify": "^1.6.0",
```

Comment:
I am using `isomorphic-dompurify` over normal `dompurify` because that doesn't run in Node and was failing in the test runner. `isomorphic-dompurify` runs in both the browser and Node.